File: Java/01.Java-Basic/04.ConditionalStatementsExercise/src/GodzillaVsKong.java
import java.util.Scanner;

public class GodzillaVsKong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double budget = Double.parseDouble(scanner.nextLine());
        int extras = Integer.parseInt(scanner.nextLine());
        double dressPrice = Double.parseDouble(scanner.nextLine());

        double decorPrice = budget * 0.1;
        double extrasDressesPrice = extras * dressPrice;
        if (extras > 150) {
            extrasDressesPrice = extrasDressesPrice - (extrasDressesPrice * 0.1);
        }
        double fimlPrice = decorPrice + extrasDressesPrice;
        if (fimlPrice > budget) {
            System.out.println("Not enough money!");
            System.out.printf("Wingard needs %.2f leva more.", fimlPrice-budget);
        } else {
            System.out.println("Action!");
            System.out.printf("Wingard starts filming with %.2f leva left.", budget-fimlPrice);
        }

    }
}

File: JavaScript/01.JavaScriptBasic/04Conditional-Statements-Exercise/src/GodzillaVsKong.js
function enoughBudget(input) {
    let budget = Number(input[0]);
    let persons = Number(input[1]);
    let pricePerPerson = Number(input[2]);

    let decorPrice = budget * 0.1;
    let dressPrice = persons * pricePerPerson;
    if (persons > 150) {
        dressPrice -= ( dressPrice * 0.1);
    }

    let totalAmount = decorPrice + dressPrice;
    if (budget >= totalAmount) {
        console.log('Action!');
        console.log(`Wingard starts filming with ${(budget - totalAmount).toFixed(2)} leva left.`);
    } else {
        console.log('Not enough money!');
        console.log(`Wingard needs ${(totalAmount - budget).toFixed(2)} leva more.`);
    }
}

enoughBudget(["20000", "120", "55.5"]);
enoughBudget(["15437.62", "186", "57.99"]);
enoughBudget(["9587.88", "222", "55.68"]);